src/browserExt/singlefile.js

```diff
@@ -138,10 +138,8 @@ Zotero.SingleFile = new function () {
 				form
 			};
 		} catch (e) {
-			// This attachment will fall back to SingleFile retrieval in the client/server
+			// This will fall back to SingleFile retrieval in the client
 			Zotero.debug("SingleFile: Error retrieving page data");
-			Zotero.debug(e);
-			throw e;
```

Comment:
Sorry, I meant only the debug line should be removed. It should still throw on error.